beancount_reds_importers/libtransactionbuilder/investments.py

```diff
@@ -11,8 +11,78 @@
 from beancount_reds_importers.libtransactionbuilder import common
 
 
+def remove_empty_subaccounts(acct):
+    """Translates 'Assets:Foo::Bar' to 'Assets:Foo:Bar'."""
+    return ':'.join(x for x in acct.split(':') if x)
+
+
+def format_raw_account(raw_acct, ot, ticker):
+    """Format raw_acct with the account name variables described in Importer.
+
+    Fills variables like {ticker}.  ot may be None if it is not available.
+    """
+    kwargs = {'ticker': ticker, '401ksrc': ''}
+    if ot is not None and hasattr(ot, 'inv401ksource'):
+        kwargs['401ksrc'] = ot.inv401ksource.title()
```

Comment:
I was thinking it'd be okay to have this here because inv401ksource is part of ofxparse, and so is a general investment field. Are you saying I should look to make this part of only the Vanguard importer?

Reply:
Got it, I'd missed that `inv401ksource` is a part of the ofx spec.